### DISCUSSION TITLE
Does `href` attribute in `UnderlineNav.Item` v2 works or supported?

### DISCUSSION THREAD

**Discussion by jhdcruz** [primer/react#2431] [Q&A] [answered]
> Not sure if this is a bug, so I asked here instead.

Based on the documentation, `UnderlineNav.Item` **v2** [renders `<a>`](https://primer.style/react/drafts/UnderlineNav2#underlinenavitem). So I presumed that using `href` in it will work. but apparently doesn't.

output renders this:
```html
<a
  href="/test"
  title="Test"
  class="..."
>
  ...
</a>
```

but clicking it doesn't redirect or anything.

----

1. **Just want to ask if this is an intentional behavior?** Or it does work and my implementation is the problem?
2. **Do I need to manually bind routing in the provided `onSelect` props?**

**Comment by lesliecdubs** [comment, marked as answer]
👋🏻 Hi there, thanks for posting! 

@broccolinisoup is this something you might be able to weigh in on?

**Reply by broccolinisoup** [reply]
Thanks @lesliecdubs for the mention! I think I can help clarifying. 

👋🏼 @jhdcruz 

That was the initial WIP implementation and was only supporting router behaviour and preventing any default redirection using `href`. This is why clicking the nav link wouldn't do any redirection as the default behaviour is prevented. However, we just added the `href` support with [this PR](https://github.com/primer/react/pull/2425) 🙌🏼  It should be available in the next release or the open PR's release version can be used as well in the meantime.

> Do I need to manually bind routing in the provided onSelect props?

You can do that too 🙌🏼  You can check the open PR's [documentation](https://primer-7f80a98c95-13348165.drafts.github.io/drafts/UnderlineNav2#underlinenavitem) for more info.

**Reply by jhdcruz** [reply]
Thanks for answering my questions.

My bad for not noticing the PR :sweat_smile:
